readthedocs/settings/base.py

```diff
@@ -125,6 +125,8 @@ def SESSION_COOKIE_SAMESITE(self):
     RTD_STABLE_VERBOSE_NAME = 'stable'
     RTD_CLEAN_AFTER_BUILD = False
     RTD_MAX_CONCURRENT_BUILDS = 4
+    RTD_BUILDS_MAX_RETRIES = 25
```

Comment:
Is this a reasonable default? This probably isn't the place to change it, but it feels like a lot.

Reply:
I thought about this as well. I think you are right that 25 as default for _all retry cases_ is not perfect and we should have a lower number for other cases. However, right now, the only situation where we perform a retry is the concurrency limit reached, as far as I can tell.

But we should expand this to be:

- `RTD_BUILDS_MAX_RETRIES=5`
- `RTD_BUILDS_CONCURRENT_LIMIT_MAX_RETRIES=25`

or similar in case we need the default value for other cases.